RU: add links to redis university courses/learning paths (#1093)
redis/docs: content/operate/rc/databases/monitor-performance.md

To update alert settings for one or more team members, select **Access Management** from the Redis Cloud console menu and then select the **Team** tab. For details, see [Access management]({{< relref "/operate/rc/security/access-control/access-management" >}}).

If you subscribe to Redis Cloud through a Platform-as-a-Service (PaaS) provider (such as Heroku), you will need to review your provider's documentation for help managing your team.

## Continue learning with Redis University

> See the [Redis Cloud monitoring overview course](https://university.redis.io/course/sdm9uvjmd0iedp) for modules.

Reviewer: ```suggestion
See the [Redis Cloud monitoring overview course](https://university.redis.io/course/sdm9uvjmd0iedp) to learn more.
```
